docs/set-minimal-iso.md

```diff
@@ -0,0 +1,54 @@
+# Generating a minimal ISO
+The default discovery ISO generated by the Assisted Installer is a complete RHCOS ISO with a custom ignition file.
+
+Serving this large file through OOBM (e.g. BMC, iDRAC) can sometimes lead to unexpected issues.
+
+To overcome this, the assisted service offers an option to generate a smaller ISO file that dynamically downloads the rest of the RHCOS rootfs via the Internet.
+
+The following script provides an example of how to use the Assisted Installer API in order to set the type of the discovery ISO to "minimal-iso".
+
+For more information about the API and its various authentication methods, see [this document](cloud.md).
+
+# Set discovery ISO type
+```
+#!/bin/bash
+
+set -euo pipefail
+
+if [ -z ${SSH_KEY+x} ]; then
+	echo 'Please set SSH_KEY to your SSH public key.'
+	echo 'For example: export SSH_KEY=$(cat ~/.ssh/id_rsa.pub)'
+	exit 1
+fi
+
+if ! ocm token 2>/dev/null >/dev/null; then
+    echo "Failed to run 'ocm token' command, please see the assisted_service/docs/cloud.md doc for authentication information"
+    exit 1
+fi
+
+## User specific configuration <-----------
+TOKEN=$(ocm token)
+OCM_API_ENDPOINT="https://api.openshift.com/api/" 
+if true; then
+    echo "Don't forget to modify the script with your cluster ID, delete this warning after you did"
+    exit 1
+fi
+CLUSTER_ID="6de7bb1a-c4ee-4010-92dc-140957c6537f" # Copy from Assisted Installer URL
```

Comment:
Why not just use another env var for this like `SSH_KEY`?

Reply:
Just copied from the other doc and got lazy, will fix